test/test_contrib.py

```diff
@@ -576,12 +576,17 @@ class TestFaceDetection:
     def test_smoke(self, device, dtype):
         assert kornia.contrib.FaceDetector().to(device, dtype) is not None
 
-    def test_valid(self, device, dtype):
+    @pytest.mark.parametrize("batch_size", [1, 2, 4])
+    def test_valid(self, batch_size, device, dtype):
         torch.manual_seed(44)
-        img = torch.rand(1, 3, 320, 320, device=device, dtype=dtype)
+        img = torch.rand(3, 320, 320, device=device, dtype=dtype)
```

Comment:
```suggestion
        img = torch.rand(batch_size, 3, 320, 320, device=device, dtype=dtype)
```
optional; more clean than stacking

Reply:
totally agree, I did a mess XD